client/ayon_traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -835,6 +895,23 @@ def _create_instances_from_csv_data(self, csv_dir: str, filename: str):
                 "prepared_data_for_repres": []
             }
 
+            if product_item.has_promised_context:
+                hierarchy, _ = folder_path.rsplit("/",1)
+                families.append("csv_ingest_shot")
```

Comment:
happy to discuss this one, we kind of need something to trigger the `collect hierarchy` and `extract hierarchy to AYON` plugins but cannot use `clip` nor `shot` cause we do not have any OTIO data here.

Reply:
Why we can't use `shot`?

Reply:
Shot is expecting otioClip at `ayon-traypublisher\client\ayon_traypublisher\plugins\publish\collect_shot_instances.py`

Reply:
https://github.com/ynput/ayon-core/pull/1010#issuecomment-2479069045